### PR TITLE
build: remove custom webpack

### PR DESCRIPTION
**Pull request by HyperLife1119**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

此 PR 基于 #8552。

通过此 PR，ZORRO 不再自定义 webpack 配置，这有利于更快的跟随 Angular 的版本更新，不必等待 @angular-builders/custom-webpack 包更新。

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/style/testing.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// This file is only used to set the @root-entry-name variable in a test environment.
+
+@import './default.less';
+// @import './variable.less';
```
Comment (HyperLife1119) [inline components/style/testing.less:4]: 当测试中需要导入样式时，必须首先导入该样式文件。该文件用于在测试中填充 @root-entry-name 变量。

**Review by Laffery — CHANGES_REQUESTED** [review]
previous pr  #8552 was merged, please rebase

File: components/style/testing.less (hunk 1 above)
Comment (Laffery) [reply components/style/testing.less:4]: 最后一行可以删掉吧👀

Comment (HyperLife1119) [reply components/style/testing.less]: 可以了

**Review by Laffery — APPROVED** [review]
LGTM

**Pull request merged**